google-cloud-spanner/lib/google/cloud/spanner/lar_headers.rb

```diff
@@ -0,0 +1,79 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+module Google
+  module Cloud
+    module Spanner
+      ##
+      # @private Helper module to access header values
+      # for Leader Aware Routing
+      module LARHeaders
+        module ClassMethods
```

Comment:
Consider just wrapping these in `class << self` rather than creating and extending a ClassMethods module. The ClassMethods module isn't really needed here, especially since this is an internal module.

Reply:
Done.